### PR TITLE
feat(state): add sdk initialization status & access token validity flags

### PR DESCRIPTION
**Pull request by mhamann**

### CODE REVIEW

File: android/src/main/java/io/rownd/android/models/domain/Auth.kt [hunk 1]
```diff
@@ -24,6 +25,17 @@ data class AuthState @OptIn(ExperimentalSerializationApi::class) constructor(
...
 ) {
+    val isAccessTokenValid: Boolean
+        get() {
+            if (accessToken == null) {
+                return false
+            }
+
+            val jwt = JWT(accessToken)
+
+            return !jwt.isExpired(0)
```
Comment (BobbyRadford) [inline android/src/main/java/io/rownd/android/models/domain/Auth.kt:36]: 0?

File: android/src/main/java/io/rownd/android/models/repos/StateRepo.kt [hunk 2]
```diff
@@ -79,12 +86,14 @@ object StateRepo {
         dataStore = ds
 
         // Re-inflate store from persistence
-        CoroutineScope(Dispatchers.IO).async {
+        CoroutineScope(Dispatchers.IO).launch {
```
Comment (BobbyRadford) [inline android/src/main/java/io/rownd/android/models/repos/StateRepo.kt:89]: launch v async?

File: android/src/main/java/io/rownd/android/models/domain/Auth.kt (hunk 1 above)
Comment (mhamann) [reply android/src/main/java/io/rownd/android/models/domain/Auth.kt:36]: leeway

File: android/src/main/java/io/rownd/android/models/repos/StateRepo.kt (hunk 2 above)
Comment (mhamann) [reply android/src/main/java/io/rownd/android/models/repos/StateRepo.kt:89]: fire and forget vs return value

**Review by BobbyRadford — APPROVED** [review]

**Pull request merged**